clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -1547,6 +1553,29 @@ public String toString() {
         }
     }
 
+    private class ConsumerCoordinatorMetrics {
```

Comment:
This is to restore the original private class as it doesn't make a lot of sense to reuse this in the new consumer.

Reply:
I split the implementation into two classes because there's no need to pass the ref of this entire object to the request manager just for the commitSensor (see addCommitSensor method).  Instead, I think it would be a lot easier to pass the Metrics object to the manager and create their own sensors (essentially these metrics objects just hold a bunch of sensors referenced from Metrics).